backend/private-graph/graph/schema.resolvers.go

```diff
@@ -443,7 +443,7 @@ func (r *mutationResolver) CreateAdmin(ctx context.Context) (*model.Admin, error
 func (r *mutationResolver) CreateProject(ctx context.Context, name string, workspaceID int) (*model.Project, error) {
 	workspace, err := r.isAdminInWorkspace(ctx, workspaceID)
 	if err != nil {
-		return nil, nil
```

Comment:
This was a bug. There's a potential lint rule here that could catch this:
https://github.com/highlight/highlight/pull/5592#issuecomment-1583574619

Reply:
this change might have been made to omit this common 'expected' error. should be ok to return a real error here now though if we ignore them all, but might want to check that the frontend doesn't break if this returns an error

Reply:
Ah I see. I'll revert for now since this was out of scope and we can take a look in #5592.
1c1b9dc96136ed95afdcc47e3c967adff230b360